### PR TITLE
created ExceptionHandlingFilter in Security Filter chain

### PR DESCRIPTION
**Pull request by kaanaktas**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CCD-3470

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/ccd/security/filters/ExceptionHandlingFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    @Test
+    public void shouldReturn500ResponseWhenAnyExceptionExceptClientAbortExceptionThrown()
+        throws ServletException, IOException {
+        Mockito.doThrow(NullPointerException.class)
+            .when(filterChain)
+            .doFilter(Mockito.eq(request), Mockito.eq(response));
+        filter.doFilterInternal(request, response, filterChain);
+
+        assertThat(response.getStatus()).isEqualTo(500);
+    }
```
Comment (paulridout) [inline src/test/java/uk/gov/hmcts/ccd/security/filters/ExceptionHandlingFilterTest.java:58]: Is it worth checking that the expected error message is written to the log in these tests?  See RestExceptionHandlerTest class for examples of this being done.

Comment (kaanaktas) [reply src/test/java/uk/gov/hmcts/ccd/security/filters/ExceptionHandlingFilterTest.java:95]: completed!

Comment (kaanaktas) [reply src/test/java/uk/gov/hmcts/ccd/security/filters/ExceptionHandlingFilterTest.java:95]: completed!

**Review by paulridout — APPROVED** [review]
Changes look OK.

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Pull request merged**